addons/website/models/website_menu.py

```diff
@@ -116,11 +116,12 @@ def unlink(self):
     def _compute_visible(self):
         for menu in self:
             visible = True
-            if (menu.page_id and not menu.user_has_groups('base.group_user')
-                and (not menu.page_id.sudo().is_visible
-                     or (not menu.page_id.view_id._handle_visibility(do_raise=False)
-                         and menu.page_id.view_id._get_cached_visibility() != "password"))):
-                visible = False
+            if menu.page_id and not menu.user_has_groups('base.group_user'):
+                page_id = menu.page_id.sudo()
```

Comment:
`page_id` was a bad name, it sounds like its holding an ID, but it's even more wrong now, it's holding a record in sudo
```suggestion
                page_sudo = menu.page_id.sudo()
```